core/src/components/refresher/refresher.tsx

```diff
@@ -106,7 +106,7 @@ export class Refresher {
       'passive': false,
       'direction': 'y',
       'threshold': 0,
-      'attachTo': 'window'
+      'attachTo': this.el.parentElement
```

Comment:
you can use 'parent' here

Reply:
This wasn't working in my tests. When setting to `parent` and inspecting the prop in dev tools, it would keep falling back to `child`